src/db/database.ts

```
@@ -0,0 +1,58 @@
+                color VARCHAR(50)
+            );
+        `;
+        await this.ExecutaComando(sql);
+    }
+
+    public async ExecutaComando(sql: string, parametros: any[] = []): Promise<any> {
+        try {
+            const [rows] = await this.pool.promise().query(sql, parametros);
+            return rows;
+        } catch (error) {
+            throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)